.github/workflows/docker-build.yml

```diff
@@ -103,11 +103,14 @@ jobs:
           ref: ${{ inputs.nightly_tag_main || github.ref }}
           persist-credentials: true
 
+      - name: "Setup Environment"
+        uses: ./.github/actions/setup-uv
```

Comment:
Not sure if this is necessary since we aren't installing anything

Reply:
it is, we use uv to pull the deps

Reply:
I'm digging up the faliure that this was causing hang on

Reply:
this was the commit https://github.com/langflow-ai/langflow/commit/98e55a419c5682e36fa1ecc7c386712d211679ff#diff-3414847e2ad632333f775cabb810f0dc0df61a570365df34750a08b00912fe82R155

the problem was runnuing $(uv tree ...)

here is where it blew up https://github.com/langflow-ai/langflow/actions/runs/11578275405/job/32234096478#step:6:11

Reply:
Ah, right, good catch